diagnostic.md

```diff
@@ -92,7 +114,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Yellow, sans-serif, cursive
```

Comment:
.5/1 The inline cursive font-family has more specificity than the sans-serif value in the css document. So the font-family will just be cursive. (Also, it can't be more than one font-family at once.)